builtin/providers/aws/resource_aws_elb_load_balancer_policy.go

```diff
@@ -0,0 +1,350 @@
+		pair["value"] = *a.AttributeValue
+		if (*policyTypeName == "SSLNegotiationPolicyType") && (*a.AttributeValue == "false") {
+			continue
+		}
+		attributes = append(attributes, pair)
+	}
+
+	d.Set("policy_name", policyName)
+	d.Set("policy_type_name", policyTypeName)
+	d.Set("load_balancer_name", loadBalancerName)
+	d.Set("policy_attribute", attributes)
+
+	return nil
+}
+
+func resourceAwsLoadBalancerPolicyUpdate(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+	reassignments := Reassignment{}
+
+	loadBalancerName, policyName := resourceAwsLoadBalancerPolicyParseId(d.Id())
+
+	assigned, err := resourceAwsLoadBalancerPolicyAssigned(policyName, loadBalancerName, elbconn)
+	if err != nil {
+		return fmt.Errorf("Error determining assignment status of Load Balancer Policy %s: %s", policyName, err)
+	}
+
+	if assigned {
+		reassignments, err = resourceAwsLoadBalancerPolicyUnassign(policyName, loadBalancerName, elbconn)
+		if err != nil {
+			return fmt.Errorf("Error unassigning Load Balancer Policy %s: %s", policyName, err)
+		}
+	}
+
+	request := &elb.DeleteLoadBalancerPolicyInput{
+		LoadBalancerName: aws.String(loadBalancerName),
+		PolicyName:       aws.String(policyName),
+	}
+
+	if _, err := elbconn.DeleteLoadBalancerPolicy(request); err != nil {
+		return fmt.Errorf("Error deleting Load Balancer Policy %s: %s", d.Id(), err)
+	}
+
+	err = resourceAwsLoadBalancerPolicyCreate(d, meta)
+
+	for _, listenerAssignment := range reassignments.listenerPolicies {
+		if _, err := elbconn.SetLoadBalancerPoliciesOfListener(listenerAssignment); err != nil {
+			return fmt.Errorf("Error setting LoadBalancerPoliciesOfListener: %s", err)
+		}
+	}
+
+	for _, backendServerAssignment := range reassignments.backendServerPolicies {
+		if _, err := elbconn.SetLoadBalancerPoliciesForBackendServer(backendServerAssignment); err != nil {
+			return fmt.Errorf("Error setting LoadBalancerPoliciesForBackendServer: %s", err)
+		}
+	}
+
+	return nil
+}
+
+func resourceAwsLoadBalancerPolicyDelete(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	loadBalancerName, policyName := resourceAwsLoadBalancerPolicyParseId(d.Id())
+
+	assigned, err := resourceAwsLoadBalancerPolicyAssigned(policyName, loadBalancerName, elbconn)
+	if err != nil {
+		return fmt.Errorf("Error determining assignment status of Load Balancer Policy %s: %s", policyName, err)
+	}
+
+	if assigned {
+		_, err := resourceAwsLoadBalancerPolicyUnassign(policyName, loadBalancerName, elbconn)
+		if err != nil {
+			return fmt.Errorf("Error unassigning Load Balancer Policy %s: %s", policyName, err)
+		}
+	}
+
+	request := &elb.DeleteLoadBalancerPolicyInput{
+		LoadBalancerName: aws.String(loadBalancerName),
+		PolicyName:       aws.String(policyName),
+	}
+
+	if _, err := elbconn.DeleteLoadBalancerPolicy(request); err != nil {
+		return fmt.Errorf("Error deleting Load Balancer Policy %s: %s", d.Id(), err)
+	}
+	return nil
```

Comment:
addressed in https://github.com/hashicorp/terraform/pull/7458/commits/4f07b924479759b47ab6df4baa906bbee5b75371